CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 9.1.1
```

Comment:
9.1.1 or 9.1.0 below in the version is the correct version?

Reply:
LOL, fixing that

Reply:
Thanks for the catch! Fixed now